src/encoded/static/components/vis_defines.js

```diff
@@ -204,6 +204,23 @@ export const visSortBrowsers = browsers => (
 );
 
 
+/**
+ * Map of browser to display name.
+ */
+const browserNameMap = {
+    UCSC: 'UCSC',
+    hic: 'Juicebox',
+    Ensembl: 'Ensembl',
+};
```

Comment:
It would be very convenient if JavaScript guaranteed the order of `Object.keys` because then you could use the object for `browserOrder` and not have an array and an object with repeated information - just so that adding a browser wouldn't require remembering to add it to both the array and the object. Oh well!